src/main/java/com/puppycrawl/tools/checkstyle/api/FileText.java

```diff
@@ -80,7 +80,7 @@
     /**
      * The full text contents of the file.
      */
-    private final String fullText;
+    private String fullText;
```

Comment:
what about a package-private constructor instead so this can stay `final`?

Reply:
The test assigns a value. Maybe this can be handled with another constructor.